dev/reference/security/role-based-access-control.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Role-Based Access Control
+summary: This document introduces TiDB RBAC operations and implementation.
+category: user guide
+---
+
+# Role-Based Access Control
+
+The implementation of TiDB's role-based access control (RBAC) system is similar to that of MySQL. TiDB is compatible with most RBAC syntax of MySQL.
+
+This document introduces TiDB RBAC-related operations and implementation.
+
+> **Warning:**
+>
+> The feature of RBAC is still in the testing phase. Do not use it in the production environment.
+
+## RBAC operations
+
+A role is a collection of a series of privileges. You can do the following operations:
+
+- Create a role.
+- Delete a role.
+- Grant a privilege to a role.
+- Grant a role to another user. That user can obtain the privileges involved in the role, after enabling the role.
+
+### Create a role
+
+For example, you can use the following statement to create roles `r_1` and `r_2`:
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE ROLE `r_1`@`%`, `r_2`@`%`;
+```
+
+For the role naming format and rule, see [TiDB User Account Management](/reference/security/user-account-management.md).
+
+Roles are stored in the `mysql.user` table. The name of the role you are trying to create must be unique; otherwise, an error is reported.
+
+To create a role, you need the `CREATE ROLE` or `CREATE USER` privilege.
+
+### Delete a role
+
+For example, you can use the following statement to delete roles `r_1` and `r_2`:
+
+{{< copyable "sql" >}}
+
+```sql
+DROP ROLE `r_1`@`%`, `r_2`@`%`;
+```
+
+This operation deletes role records in the `mysql.user` table, removes related records in the authorization table, and terminates role-related authorization relationship.
+
+To delete a role, you need the `DROP ROLE` or `DROP USER` privilege.
+
+### Grant a privilege to a role
+
+The operation of granting a privilege to a role is the same with that of granting a privilege to a user. For details, see [TiDB Privilege Management](/reference/security/privilege-system.md).
+
+For example, you can use the following statement to grant the `xxx` user the privilege to read the `test` database:
```

Comment:
Can we use more meaningful examples than `xxx`? Since it is granting `SELECT` may I suggest naming the role "analyst"?

Reply:
Good idea!

Reply:
OK